libs/fluid-elements/checkbox/src/lib/checkbox.spec.ts

```diff
@@ -218,6 +218,27 @@ describe('Fluid checkbox', () => {
     });
   });
 
+  describe('value', () => {
+    it('should set the default value to on', async () => {
+      const shadowedInput = fixture.shadowRoot?.querySelector('input');
+      expect(shadowedInput?.getAttribute('value')).toBe('on');
+    });
+
+    it('should set the value to customvalue when the property is set', async () => {
+      fixture.value = 'customvalue';
+      await tick();
+      const shadowedInput = fixture.shadowRoot?.querySelector('input');
+      expect(shadowedInput?.getAttribute('value')).toBe('customvalue');
+    });
+
+    it('should set the value to customvalue when the attribute is set', async () => {
```

Comment:
Please add a test case that is setting the value via clicking? Then you cover all the scenarios for how to set the value.

Reply:
Value of a checkbox cannot be set vie clicking. The value does not represent the state of the checkbox (as in checked / unchecked). Example from mdn

> A DOMString representing the value of the checkbox. This is never seen on the client-side, but on the server this is the value given to the data submitted with the checkbox's name. Take the following example:

```html
<form>
  <div>
    <input type="checkbox" id="subscribeNews" name="subscribe" value="newsletter">
    <label for="subscribeNews">Subscribe to newsletter?</label>
  </div>
  <div>
    <button type="submit">Subscribe</button>
  </div>
</form>
```